eth/accesslist_test.go

```diff
@@ -0,0 +1,30 @@
+package eth_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/INFURA/go-ethlibs/eth"
+)
+
+func TestNewAccessListFromRLP(t *testing.T) {
+	src := eth.AccessList{
+		eth.AccessListEntry{
+			Address: "0x0000000000000000000000000000000000001337",
+			StorageKeys: []eth.Data32{
```

Comment:
Maybe also test with multiple elements? Or is this case intended to be tested by `TestTransactionTypeAccessList_Populated`?

Reply:
Added more cases in c9fe053.